bundles/org.openhab.binding.luxtronikheatpump/README.md

```diff
@@ -0,0 +1,331 @@
+# LuxtronikHeatpump Binding
+
+This binding gives the possibility to integrate any Heatpump that is based on the Luxtronik 2 contol unit of Alpha Innotec. This includes heatpumps of:
+
+* Alpha InnoTec
+* Buderus (Logamatic HMC20, HMC20 Z)
+* CTA All-In-One (Aeroplus)
+* Elco
+* Nibe (AP-AW10)
+* Roth (ThermoAura®, ThermoTerra)
+* (Siemens) Novelan (WPR NET)
+* Wolf Heiztechnik (BWL/BWS)
+
+This binding was tested with:
+
+* Siemens Novelan LD 7
+
+_If you have another heatpump the binding works with, let us know, so we can extend the list_
+
+Note: The whole functionality is based on data that was reverse engineered, so use it at your own risk.
+
+## Supported Things
+
+This binding only supports one thing type "Luxtronik Heatpump" (heatpump).
+
+## Thing Configuration
+
+Each heatpump requires the following configuration parameters:
+
+| parameter    | required | default | description |
+|--------------|----------|---------|-------------|
+| ipAddress    | yes      |         | IP address of the heatpump |
+| port         | no       | 8889    | Port number to connect to. This should be `8889` for most heatpumps. For heatpumps using a firmware version before V1.73 port `8888` needs to be used. |
+| refresh      | no       | 300     | Interval (in seconds) to refresh the channel values. |
+| showAllChannels | no       | false    | Show all channels (even those determined as not supported) |
+
+## Channels
+
+As the Luxtronik 2 control is able to handle multiple heat pumps with different features (like heating, hot water, cooling, solar, photovoltaics, swimming pool,...), the binding has a lot channels. Depending on the heatpump it is used with, various channels might not hold any (useful) values. If `showAllChannels` is not activated for the thing, this binding will automatically try to hide channels that are not available for your heat pump. As this is done using reverse engineered parameters it might not be correct in all cases. If you miss a channel that should be available four your heat pump, you can enable `showAllChannels` for your thing, so all channels become available. Feel free to report such a case on the forum, so we can try to improve / fix that behavior.
```

Comment:
If you put each sentence in a separate line, reading the diffs would be easier.
```suggestion
As the Luxtronik 2 control is able to handle multiple heat pumps with different features (like heating, hot water, cooling, solar, photovoltaics, swimming pool,...), the binding has a lot channels. Depending on the heatpump it is used with, various channels might not hold any (useful) values. If `showAllChannels` is not activated for the thing, this binding will automatically try to hide channels that are not available for your heat pump. As this is done using reverse engineered parameters it might not be correct in all cases. If you miss a channel that should be available for your heat pump, you can enable `showAllChannels` for your thing, so all channels become available. Feel free to report such a case on the forum, so we can try to improve / fix that behavior.
```